src/gui/ChecksumTool.cpp

```diff
@@ -186,7 +186,7 @@ void ChecksumTool::LoadOnlineData() const
 
 		std::string latest_commit;
 
-		const auto checksum_path = ActiveSettings::GetPath("resources/checksums");
+		const auto checksum_path = GetPath(ActiveSettings::GetUserDataPath(), "resources/checksums");
```

Comment:
I see this all over the code changes. Why not leave the GetPath functions in the ActiveSettings class and add more functions like ActiveSettings::GetPathUserData and ActiveSettings::GetPathConfig that just call GetPath and prepend the corresponding paths? Would make for a cleaner patch since ActiveSettings.cpp doesn't get modified as much and make the code more readable in my opinion.

Reply:
There's probably some clever way to roll GetMLCPath and the various other Path functions into one. Looks like GetPath and GetMLCPath already share a lot of code. Beware of the changes in commit 168ecf8825813c570793510a0314130c3fb18385. It made some Path functions use the _utf8ToPath function and others not. If we make it all consistent it should fit really neatly I think.

Reply:
I have a PR in the works that will address this.

Reply:
Nevermind you beat me to it!

Reply:
Now all GetXPath methods can be called in a similar way as the original GetPath method.